app/forms/keywords_form.rb

```diff
@@ -19,7 +19,7 @@ def save(file)
     begin
       keyword_records = parse_keywords.map { |keyword| keyword_record(keyword) }
       # rubocop:disable Rails::SkipsModelValidations
-      Keyword.insert_all(keyword_records)
+      @keyword_ids = Keyword.insert_all(keyword_records).map { |hash| hash['id'] }
```

Comment:
Minor renaming
```suggestion
      @keyword_ids = Keyword.insert_all(keyword_records).map { |keyword| keyword['id'] }
```

Reply:
Fixed in [`36c45ec` (#45)](https://github.com/suho/google-search-ruby/pull/45/commits/36c45ec1188d351e4058ceb33a9673471bc0f433)